.github/workflows/update-dependencies.yml

```
@@ -0,0 +1,32 @@
+name: Update DEPENDENCIES file
+    
+on:
+  workflow_call:
+    inputs:
+      pom:
+        description: 'Path to pom.xml'
+        type: string
+        required: false
+        default: 'pom.xml'
+
+jobs:        
+  release-tag:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write  
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Bump versions
+      uses: eclipse-set/build/.github/actions/update-dependency-file@main
```

I guess this is an c&p error?
```suggestion
    - name: Update DEPENDENCIES
      uses: eclipse-set/build/.github/actions/update-dependency-file@main
```